src/main/java/com/treasure/hunt/ui/swing/SwingTest.java

```diff
@@ -0,0 +1,101 @@
+package com.treasure.hunt.ui.swing;
+
+import com.treasure.hunt.jts.Circle;
+import com.treasure.hunt.strategy.geom.GeometryItem;
+import com.treasure.hunt.strategy.geom.GeometryType;
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.GeometryFactory;
+import org.locationtech.jts.geom.Polygon;
+import org.locationtech.jts.geom.util.AffineTransformation;
+
+import javax.swing.*;
+import javax.swing.plaf.metal.MetalLookAndFeel;
+import java.awt.*;
+
+public class SwingTest extends JPanel {
+
+    public static void main(String[] args) {
+        JFrame frame = new JFrame();
+        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        frame.setSize(400, 400);
+        frame.setVisible(true);
+
+        GeometryPanel geometryPanel = new GeometryPanel();
+        frame.setContentPane(geometryPanel);
+
+        MetalLookAndFeel.setCurrentTheme(new TreasureHuntTheme());
+        try {
+            UIManager.setLookAndFeel(new MetalLookAndFeel());
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+        SwingUtilities.updateComponentTreeUI(frame);
+    }
+
+    public static GeometryItem[] exampleGeometryItems() {
+        GeometryFactory geometryFactory = new GeometryFactory();
+
+        Circle c1 = new Circle(new Coordinate(100, 100), 50, geometryFactory);
+        Circle c2 = new Circle(new Coordinate(150, 100), 50, geometryFactory);
+        Polygon intersection = (Polygon) c1.intersection(c2);
+
+        AffineTransformation affineTransformation = new AffineTransformation().translate(0.0, 20.0);
+
+        intersection = (Polygon) affineTransformation.transform(intersection);
+
+        Polygon c3 = (Polygon) affineTransformation.transform(c1);
+
+        GeometryType intersectionType = new GeometryType(true, Color.red, Color.yellow, true, "");
+
+        return new GeometryItem[]{
+                new GeometryItem<>(c1, new GeometryType(true, Color.green)),
+                new GeometryItem<>(c2, new GeometryType(true, Color.blue)),
+                new GeometryItem<>(c3, intersectionType),
+                new GeometryItem<>(intersection, intersectionType),
+        };
+    }
+
+    public void paint(Graphics graphics) {
+        /*Graphics2D graphics2D = (Graphics2D) graphics;
```

Comment:
Please never commit commented out code thats what we have git for, or if it is not ready dont merge